### ISSUE TITLE
Remove banner

### ISSUE THREAD

**Issue by 6temes** [rubycoco/record#1]
Hi,

As in https://github.com/csvreader/csvreader/issues/5, it would be good to remove the banner, so it does not generate noise in the condole.

https://github.com/rubycoco/record/blob/master/lib/record.rb#L14

Thanks!

**Comment by geraldb** [comment]
Good point again. Updated the code. Will get (pushed) out with the next gem version. Cheers. Prost.
